src/clients/redis/redis.client.ts

```diff
@@ -8,6 +8,8 @@ export const createRedisClient = (): RedisClient => {
   });
   redisClient.on('error', error => {
     console.error(`Redis error: ${error}`);
+    console.error('Please check your Redis instance.');
```

Comment:
What do you think about joining the console logs on just one console.error and use the \n to write in a new line. Is a minor detail hehe

Reply:
it's done, thanks!!